app/views/guests/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1><%= @guest.name %></h1>
+
+<h2>Rooms:</h2>
+<ul>
+  <% @guest.rooms.each do |room| %>
+	<li>
+		Suite: <%= room.suite %><br>
+		Nightly Rate: <%= room.rate %><br>
+		Hotel: <%= room.hotel.name %>
+	</li>
+<% end %>
+</ul>
+
+<h2>Add Room:</h2>
+<%= form_with url: "/guests/#{@guest.id}/add_room", method: :post do %>
```

Comment:
You'll want to make sure to use your block variable for the form object to create the text fields, like this: https://backend.turing.edu/module2/lessons/forms#:~:text=app/views/artists/new.html.erb
The form_tags are soft deprecated so you want to switch to using the form_with form builder syntax.